### ISSUE TITLE
browser

### ISSUE THREAD

**Issue by 0rrr** [NSSpiral/Blackb0x#156]
is there anyway i can use this jailbreak to install a browser... i want to use my old apple tv 3 to display browser based art.

thank you

**Comment by NSSpiral** [comment]
Yes, this is possible. Will have something you can use for this in the near
future.

**Comment by 0rrr** [comment]
amazing would it be possible to add a feature to rotate the screen 90° and make the browser window fullscreen without any tabs or menu?
